contracts/BinaryOptionMarketData.sol

```diff
@@ -189,6 +194,7 @@ contract BinaryOptionMarketData {
         address creator;
         IBinaryOptionMarket.Options options;
         IBinaryOptionMarket.Times times;
+        IBinaryOptionMarket.OraclePriceAndTimestamp oraclePriceAndTimestamp;
```

Comment:
This `MarketParameters` struct is for things which don't change over the lifetime of the contract, while the `MarketData` one is for those that do change. The oracle price changes, so it belongs in `MarketData`.

It's probably worth adding comments on this just above the structs to make this clear for others reading the code as well.